### DISCUSSION TITLE
Getting Error in notifier component

### DISCUSSION THREAD

**Discussion by imaskm** [quay/clair#1234] [Q&A] [answered]
Hi, I am running the clair as "combo"  and in a kubernetes cluster, clair service is logging this error:

`{"level":"error","component":"notifier/service/Init","component":"notifier/service/keyManagerInit","component":"notifier/keymanager/Manager.loop","time":"2021-04-10T20:10:08Z","time":"2021-04-10T20:10:08Z","message":"received error when bumping public key expiration"}

**Comment by ldelossa** [comment, marked as answer]
@imaskm this error can safely be ignored. It's dead code from a Quay related feature we did not move forward with. 

We are ripping out this code path and you should no longer see this error after 4.1

**Reply by imaskm** [reply]
Thanks @ldelossa
